test/scalability/README.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
```

Comment:
Will this framework be integrated in CI or is this not planned?

Reply:
Yes, but for the purpose I need to define a function that will say if the CI should fail or not. Maybe you can help me here  -- given a set of measurements stored in a set of pandas dataframes, which may or may not follow a normal distribution and may or may not have outliers, how does one determine if one version is better or worse than another?

Reply:
Depending on how you compare the data, it might make sense to remove the outliers so that they don't distort the results.

A simple approach could be to compare the median or the bounds (or upper bound) which contain x% (e.g., 2/3) of the data points. (For these, you would not need to remove outliers.)